po/zh_CN.po

```diff
@@ -1274,76 +1274,76 @@ msgstr "笛卡爾"
 #: js/block.js:3790
 #: js/blocks/ExtrasBlocks.js:622
 msgid "polar"
-msgstr ""
+msgstr "极坐标"
 
 #: js/block.js:3778
 #: js/block.js:3791
 #: js/blocks/ExtrasBlocks.js:626
 msgid "Cartesian/Polar"
-msgstr ""
+msgstr "笛卡尔/极坐标"
 
 #: js/block.js:3779
 #: js/block.js:3798
 #: js/blocks/ExtrasBlocks.js:655
 msgid "none"
-msgstr ""
+msgstr "无"
 
 #: js/block.js:3792
 #: js/blocks/ExtrasBlocks.js:631
 msgid "treble"
-msgstr ""
+msgstr "高音"
 
 #: js/block.js:3793
 #: js/blocks/ExtrasBlocks.js:635
 msgid "grand staff"
-msgstr ""
+msgstr "大五线谱"
 
 #: js/block.js:3794
 #: js/blocks/ExtrasBlocks.js:639
 msgid "mezzo-soprano"
-msgstr ""
+msgstr "中高音"
 
 #: js/block.js:3795
 #: js/blocks/ExtrasBlocks.js:643
 msgid "alto"
-msgstr ""
+msgstr " 女低音"
 
 #: js/block.js:3796
 #: js/blocks/ExtrasBlocks.js:647
 msgid "tenor"
-msgstr ""
+msgstr "男高音"
 
 #: js/block.js:3797
 #: js/utils/musicutils.js:1109
 #: js/utils/synthutils.js:80
 #: js/blocks/ExtrasBlocks.js:651
 #.TRANS: musical instrument
 msgid "bass"
-msgstr ""
+msgstr 低音""
 
 #: js/block.js:3842
 #: js/blocks.js:2600
 #: js/blocks.js:3211
-msgid "on2"
-msgstr ""
+msgid "on"
```

Comment:
The message id need to be on2 (this is in order to disambiguate between on (on event do) and (on/off)